nixos/modules/services/misc/sourcehut/service.nix

```diff
@@ -162,7 +162,7 @@ in
     postgresql = {
       database = mkOption {
         type = types.str;
-        default = "${srv}.sr.ht";
+        default = srvsrht;
```

Comment:
This is a breaking change for people upgrading who doesn't have the value set.

Maybe ensureDBOwnership should be a DB -> username mapping instead of a boolean value?

Reply:
Yes I explained my goals regarding this and mentioned my plans in the PR body.
I won't leave it like this and introduce proper warnings and keep the same defaults.

Reply:
Making this a map is a possibility. But we'd also run a high risk of just perpetuating the failure that `ensure*` is, if we start designing new DSL without proper thought going into its design.

Maybe better to fix this for now by `ensureDBOwnership = false;` and setting the owner ourselves in `preStart` or whatnot.

Reply:
Hm that's a possibility as well, I guess.